### ISSUE TITLE
Container issue: Create reusable UI using the RCL project 

### ISSUE THREAD

**Issue by Rick-Anderson** [dotnet/AspNetCore.Docs#16639]
This is the Container  issue for this doc.  Reopen each issue you are working on so the owner is notified.

PU Issues:
- [ ]  #16632
- [x] #13951
- [x] #17484
- [ ]  #18073

Not PU issues:

- [ ] RCL 3.0 walkthrough #14463

![image](https://user-images.githubusercontent.com/3605364/72834399-d51e9400-3c2c-11ea-8584-9d65e2f4946e.png)

Per `@javiercn:`

* We've never supported static web assets of libraries that are not referenced at compile time.
* Static web assets are a compile time feature with minimal runtime support for development.

For people looking to create reusable UI libraries that can be loaded dynamically and handle static content, I suggest looking into [Orchard ](https://docs.orchardcore.net/dev/) 🙂 or a similar technology.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9fd019c7-ceb7-6ef3-26ba-6b44341b6fc5
* Version Independent ID: fd116d00-0dc7-efaf-c714-34d77b345a05
* Content: [Reusable Razor UI in class libraries with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.1&tabs=visual-studio#feedback)
* Content Source: [aspnetcore/razor-pages/ui-class.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/razor-pages/ui-class.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-razorpages**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by zsmorris** [comment]
These docs are in desperate need of working examples and/or sample projects.  There seem to be scenarios where things just don't work.  The issues described in #16632 in particular.

**Comment by tbebekis** [comment]
Is there any news on the issue of using static files (js, css) from a dynamically loaded RCL?

**Comment by Rick-Anderson** [comment]
@mkArtakMSFT  please respond to this customer question. 

> Is there any news on the issue of using static files (js, css) from a dynamically loaded RCL?

**Comment by RenatoCaenaro** [comment]
> Is there any news on the issue of using static files (js, css) from a dynamically loaded RCL?

@mkArtakMSFT any news about intellisense in visual studio loaded from css inside an RCL as static file?

**Comment by MackinnonBuck** [comment]
Discussed the remaining items internally. We don't support what https://github.com/dotnet/AspNetCore.Docs/issues/16632 is asking for, so we're not going to document anything in that area. We decided that we aren't going to add documentation for https://github.com/dotnet/AspNetCore.Docs/issues/18073 unless we get considerable feedback that there is confusion in this specific area. Closing this out for now.
